### PR TITLE
GP2-388_list_of_countries_with_regions

### PR DESCRIPTION
**Pull request by bobmeredith**
To do (delete all that do not apply):

 - [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.

### CODE REVIEW

File: directory_constants/helpers.py [hunk 1]
```diff
@@ -56,3 +56,26 @@ def build_country_choices(
             country_list.append((country_item['Key'], country_item['Name']))
     country_list.sort(key=itemgetter(1))
     return country_list
+
+
+def build_country_region_choices(
```
Comment (webbyfox) [inline directory_constants/helpers.py:61]: Do we require any test for this?

File: directory_constants/helpers.py [hunk 2]
```diff
@@ -56,3 +56,26 @@ def build_country_choices(
...
+    include=[],
+    exclude=[]
+):
+
+    country_list = []
+    exclude = UAE_TERRITORIES_ISO_CODES + exclude
+    with (fixtures / 'countryorterritory.region.type.json').open('r') as f:
+        countries_and_territories_data = []
+        for item in json.loads(f.read()):
+            iso2 = item.get('Country or Territory: ISO 2 code')
```
Comment (depsiatwal) [inline directory_constants/helpers.py:72]: why ISO 2 any particular reason ?
and not ISO 3 ?

**Review by depsiatwal — COMMENTED** [review]
curious why ISO2

File: directory_constants/helpers.py (hunk 1 above)
Comment (bobmeredith) [reply directory_constants/helpers.py:61]: I guess so, but I couldn't see how to even run any tests we have.
I'll try to get it going and add a test

**Comment by bobmeredith** [comment]
It's a shorter form of the original key name 'Country or Territory: ISO 2 code' as in 2 character iso code rather than 3 character.

File: directory_constants/helpers.py (hunk 1 above)
Comment (bobmeredith) [reply directory_constants/helpers.py:61]: Added a test.

**Review by depsiatwal — APPROVED** [review]

**Pull request merged**